<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAlignLeft">
       <property name="text">
        <string>居左</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/images/508.bmp</normaloff>:/icons/images/508.bmp</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAlignCenter">
       <property name="text">
        <string>居中</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/images/510.bmp</normaloff>:/icons/images/510.bmp</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAlignRight">
       <property name="text">
        <string>居右</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/images/512.bmp</normaloff>:/icons/images/512.bmp</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnFontBold">
       <property name="text">
        <string>粗体</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/images/500.bmp</normaloff>:/icons/images/500.bmp</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFontItalic">
       <property name="text">
        <string>斜体</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/images/502.bmp</normaloff>:/icons/images/502.bmp</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFontUnderline">
       <property name="text">
        <string>下划线</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/images/504.bmp</normaloff>:/icons/images/504.bmp</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkReadOnly">
        <property name="text">
         <string>ReadOnly</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkEnabled">
        <property name="text">
         <string>Enabled</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkClearEnabled">
        <property name="text">
         <string>ClearButtonEnabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="radioBlack">
        <property name="text">
         <string>Black</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioRed">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBlue">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>测试用的文字</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
